modules/cluster-and-ha-management/pages/set-up-crr.adoc

```diff
@@ -66,6 +66,9 @@ $ gadmin config set System.CrossRegionReplication.TopicPrefix Primary
 $ gadmin config apply -y
 $ gadmin init kafka -y
 $ gadmin restart all -y
+
+# Start mirrormaker and begin cross-region replication
```

Comment:
We don't need L67 & L68 right?

Reply:
While the `gadmin crr` commands do not require a restart, services such as GSQL and GPE need to be restarted once to be in the DR state

Reply:
Why is L67 needed? If we only need to restart GPE/GSQL/GSE, we should change to that only instead of all.

Reply:
Yes, L67 is unnecessary, I have verified it through e2e.
https://github.com/tigergraph/cqrs/pull/2844/files